ui/ui-frontend-common/src/app/modules/object-editor/required.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { EditObject } from './models/edit-object.model';
+
+@Pipe({ name: 'required' })
+export class RequiredPipe implements PipeTransform {
+  transform(value: string, editObject: EditObject) {
+    if (editObject.required) return `${value}*`;
+
+    return value;
```

Comment:
J'aime bien les syntaxes plus concises du style :
```suggestion
    return `${value}${editObject.required ? '*' : ''}`;
```
mais c'est bon aussi.